packages/core/src/types.tsx

```diff
@@ -378,7 +395,7 @@ export type RouteConfig<
     | ScreenOptions
     | ((props: {
         route: RouteProp<ParamList, RouteName>;
-        navigation: NavigationHelpersCommon<ParamList>;
+        navigation: any;
```

Comment:
hm?

Reply:
screens receive a more complex navigation type (as in the README), and it's not really possible to type it properly here. that's why it's `any`. users need to type it manually like in README.